src/index.html

```diff
@@ -9,6 +9,80 @@
     <link rel="stylesheet" href="./style.css">
   </head>
   <body>
-    <h1>Moyo header</h1>
+    <header class="header">
+      <img
+        src="/images/logo.png"
+        alt="logo"
+        class="image"
+      >
+      <nav class="nav">
+        <ul class="ul">
+          <li class="li">
+            <a
+              href="#"
+              class="a"
+            >
+            apple
+            </a>
+          </li>
+          <li class="li">
+            <a
+              href="#"
```

Comment:
add corresponding values for href attribute for all links
```suggestion
              href="#samsung"
```